adapters/axonix/axonix.go

```diff
@@ -0,0 +1,135 @@
+package axonix
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type AxonixAdapter struct {
```

Comment:
This can simply be `type adapter struct {`, as if you were to reference it from outside this package, it would be `axonix.AxonixAdapter`, which is redundant. Also the adapters are only used as interfaces, so it need not be exported. The `Builder()` method provides the hook to access your adapter implementation.